tests/test_sentinelhub_masking.py

```diff
@@ -103,15 +104,23 @@ def config_fixture():
     ],
     ids=["basic", "resolution", "maxcc,downsampling"],
 )
-def test_cloud_mask_request(input_params, mask_shape, clm_stats, clp_stats, config):
+def test_cloud_mask_request(
+    input_params: Dict[str, Any],
+    mask_shape: Tuple[int, int, int],
+    clm_stats: Dict[str, float],
+    clp_stats: Dict[str, float],
+    config: SHConfig,
+) -> None:
     """Integration tests for CloudMasKRequest class that interacts with Sentinel Hub service"""
     request = CloudMaskRequest(config=config, **input_params)
 
     masks = request.get_cloud_masks()
-    assert_statistics_match(masks, exp_shape=mask_shape, exp_dtype=np.int8, **clm_stats, abs_delta=1e-4)
+    assert_statistics_match(masks, exp_shape=mask_shape, exp_dtype=np.dtype(np.int8), **clm_stats, abs_delta=1e-4)
```

Comment:
I see we need to correct `assert_statistics_match`

Reply:
going to open in issue